src/app/delegate/api/register/route.test.ts

```diff
@@ -52,25 +52,19 @@ it('successfuly calls PR creation', async () => {
   `);
   expect(isAddressAnAccountMock).toHaveBeenCalledTimes(1);
   expect(createDelegationPRMock).toHaveBeenCalledTimes(1);
-  expect(createDelegationPRMock.mock.lastCall).toMatchInlineSnapshot(`
-    [
-      {
-        "address": "0x6A5DD51Da29914e8659b9CC354B414f30c7692c4",
-        "description": "Delegatee description",
-        "image": File {
-          Symbol(kHandle): Blob {},
-          Symbol(kLength): 3801,
-          Symbol(kType): "image/jpeg",
-        },
-        "interests": "blockchain, NFTs",
-        "name": "Delegatee name",
-        "signature": "0x52a3c23ef6c6817691872b77615ef30927453d641acd8c607de458d39e581bcd5411f723102640897af151644086abf4f3a9baf216d684d784194aef2c6730be1c",
-        "twitterUrl": "https://example.com/x",
-        "verificationUrl": "https://example.com/verification",
-        "websiteUrl": "https://example.com",
-      },
-    ]
-  `);
+  const request = createDelegationPRMock.mock.lastCall![0];
+  expect(request.address).toBe('0x6A5DD51Da29914e8659b9CC354B414f30c7692c4');
+  expect(request.description).toBe('Delegatee description');
+  expect(request.image).toBeInstanceOf(File);
+  expect((request.image as File).arrayBuffer()).resolves.toEqual(imageBuffer.buffer);
+  expect(request.interests).toBe('blockchain, NFTs');
+  expect(request.name).toBe('Delegatee name');
+  expect(request.signature).toBe(
+    '0x52a3c23ef6c6817691872b77615ef30927453d641acd8c607de458d39e581bcd5411f723102640897af151644086abf4f3a9baf216d684d784194aef2c6730be1c',
+  );
+  expect(request.twitterUrl).toBe('https://example.com/x');
+  expect(request.verificationUrl).toBe('https://example.com/verification');
+  expect(request.websiteUrl).toBe('https://example.com');
```

Comment:
the reason I'd rather leave the snapshot is that if we add new fields to the form then it has to be added here explicitly

and we're missing checking for the file properties like mime type and size in this case

Reply:
which timestamp?

Reply:
the file's created at